### DISCUSSION TITLE
More Profile Social Links

### DISCUSSION THREAD

**Discussion by gabrielmoris** [dailydotdev/daily#768] [Feature Requests]
Feature Request: Add links to Linkedin, gitlab, possibility to add our own extra sites, adding inputs with name and website

**Comment by idoshamun** [comment]
Hi @gabrielmoris,

Thanks for the suggestion. I'll log it in our backlog :)

**Reply by gabrielmoris** [reply]
@idoshamun Thank you for the reply and for the web/app it makes very easy to follow find and read articles 🙂

**Comment by andypiper** [comment]
Adding to this, I'd like to be able to add a link to Mastodon since I no longer use Twitter.

**Reply by castarco** [reply]
Same. I arrived here because I was a bit annoyed that I have to add metadata tags associated to Twitter to my blog posts if I want Daily.dev to make de connection. It would be really nice if there was a way to use my Fediverse/Mastodon identity, or claim ownership of my domain by other means (like exposing a static file with a random name and a random value inside it that matches the expectations of Daily.dev).

I understand that this takes effort, so please don't picture me as requiring anything as if it was my divine right, I know development teams have to be juggling priorities all the time.

**Comment by danielpilich** [comment]
Hi, 

It seemed to me that the profile is still missing a social links of several platforms. For me, it would be great if I had boxes to add my Twitch and Itch.io profile, since I share my code work on the first page and share already released games on the second. 

What do you think about this?

**Reply by andypiper** [reply]
So far, the dailydotdev people have shown no interest in expanding this, after 2 years of asking. 🤷🏻

**Comment by idoshamun** [comment]
I forgot to update here that we recently added a lot more social links. Please have a look
https://dly.to/LGQolzRZGEc
